doc/setup/quickstart_cmake/index.md

```diff
@@ -16,47 +16,63 @@ Requirements:
   - cmake >= 3.4
   - git
 
-SeqAn3 requires a compiler with full C++20 support *or* GCC >= 7. Current versions of LLVM/Clang and VisualStudio/MSVC are **not supported**.
-We will briefly explain how to install GCC-7 on some popular operating systems, but we recommend using the latest version of GCC available. For more information refer to your operating system's documentation.
+## Installing GCC
 
-**Ubuntu >= 18.04**
-```
+SeqAn requires a compiler with full C++20 support *or* GCC >= 7. Current versions of LLVM/Clang and VisualStudio/MSVC
+are **not yet supported**.
+We will briefly explain how to install GCC-10 (or the latest GCC if such an option is available) on some popular
+operating systems. We recommend using the latest version of GCC available. For more information, refer to your
+operating system's documentation.
+
+\startcollapsible{Linux-based}
+
+#### Ubuntu >= 18.04
+```bash
 sudo apt install g++
 ```
-**Ubuntu < 18.04**
-```
+
+#### Ubuntu < 18.04
+```bash
 sudo add-apt-repository ppa:ubuntu-toolchain-r/test
 sudo apt update
-sudo apt install g++-7
-```
-**MacOS** using [Homebrew](https://brew.sh/)
-```
-brew install gcc@7
+sudo apt install g++-10
 ```
 
-**MacOS** using [MacPorts](https://www.macports.org/)
-```
-sudo port install gcc-7
+#### Using [conda](https://conda.io)
+    To avoid interference with system packages, we recommend creating a new environment when using conda.
+```bash
+conda create -n conda_gcc_env -c conda-forge gcc_linux-64
+conda activate conda_gcc_env
 ```
+    This will put GCC in a separate environment `conda_gcc_env` which can be activated via `conda activate conda_gcc_env`
+    and deactivated via `conda deactivate`.
 
-**Linux** using [conda](https://conda.io)
-```
-conda create -n gcc7 -c quantstack gcc-7 libgcc-7
-conda activate gcc7
-```
-This will put GCC-7 in a separate environment called `gcc7` which can be activated via `conda activate gcc7` and deactivated via `conda deactivate gcc7`.
+\endcollapsible
+
+\startcollapsible{macOS}
```

Comment:
Maybe add a section for windows with hints to the WSL or to `gitpod.com`.

And a section for try it out in your browser, which points to gitpod.com.